deploy/kubernetes/complete-demo.yaml

```diff
@@ -491,12 +625,17 @@ spec:
     metadata:
       labels:
         name: rabbitmq
+      annotations:
+        prometheus.io/scrape: "false"
     spec:
       containers:
       - name: rabbitmq
-        image: rabbitmq:3.6.8
+        image: rabbitmq:3.6.8-management
```

Comment:
please take a look at the queue-master deployment, is not able to reach the ready status